build.gradle.kts

```diff
@@ -65,6 +65,7 @@ subprojects {
 
   tasks.withType<ShadowJar> {
     relocate("com.fasterxml.jackson", "org.komamitsu.thirdparty.jackson")
+    relocate("org.msgpack.jackson.", "org.komamitsu.thirdparty.msgpack.jackson")
```

Comment:
> `org.msgpack.jackson.`

The tailing dot is needed?

Reply:
True. I will fix it.

```
javap -classpath fluency-core/build/libs/fluency-2.6.3-SNAPSHOT-shadow.jar org.komamitsu.thirdparty.msgpack.jacksondataformat.MessagePackFactory
Compiled from "MessagePackFactory.java"
public class org.komamitsu.thirdparty.msgpack.jacksondataformat.MessagePackFactory extends org.komamitsu.thirdparty.jackson.core.JsonFactory {
```

Reply:
Updated.
https://github.com/komamitsu/fluency/pull/393/commits/9a6b0a2892c5ecd8b814b78acada7b52e30c68c9

I have confirmed relocated classes have the expected FQCN! Thanks for the review.

```
javap -classpath fluency-core/build/libs/fluency-2.6.3-SNAPSHOT-shadow.jar org.komamitsu.thirdparty.msgpack.jackson.dataformat.MessagePackFactory
Compiled from "MessagePackFactory.java"
public class org.komamitsu.thirdparty.msgpack.jackson.dataformat.MessagePackFactory extends org.komamitsu.thirdparty.jackson.core.JsonFactory {
```